### PR TITLE
Manual notifications

### PR DESCRIPTION
**Pull request by TheSylence**
Allow user to be notified about any build they want to.

### CODE REVIEW

File: BuildNotifications/ViewModel/Tree/BuildNodeViewModel.cs [hunk 1]
```diff
@@ -68,6 +68,16 @@ public bool DoShortHighlight
             }
         }
 
+        public bool EnableManualNotification
```
Comment (BerndNK) [inline BuildNotifications/ViewModel/Tree/BuildNodeViewModel.cs:71]: Same rename here

File: BuildNotifications/ViewModel/Tree/BuildNodeViewModel.cs [hunk 2]
```diff
@@ -134,22 +146,32 @@ public double ProgressToDisplay
...
-        protected override BuildStatus CalculateBuildStatus() => _buildStatus;
+        protected override BuildStatus CalculateBuildStatus()
+        {
+            return _buildStatus;
+        }
 
-        protected override DateTime CalculateChangedDate() => _changedDate;
+        protected override DateTime CalculateChangedDate()
+        {
+            return _changedDate;
```
Comment (BerndNK) [inline BuildNotifications/ViewModel/Tree/BuildNodeViewModel.cs:163]: Why change this? Wasn't the expression body better readable?

**Review by BerndNK — APPROVED** [review]
Nice. Didn't think it would be that easy to be honest. Elegantly solved in the UI!

File: BuildNotifications/ViewModel/Tree/BuildNodeViewModel.cs (hunk 2 above)
Comment (TheSylence) [reply BuildNotifications/ViewModel/Tree/BuildNodeViewModel.cs:163]: Good question. Seems like my Code Cleanup profile was not configured properly. Fixed.

**Pull request merged**